miner/worker.go

```diff
@@ -1095,6 +1102,95 @@ func (w *worker) commitBundle(env *environment, txs types.Transactions, interrup
 	return nil
 }
 
+func (w *worker) commitTransactionWrapper(env *environment, interrupt *int32, tx *types.Transaction, coalescedLogs []*types.Log) error {
```

Comment:
curious if this is copied from somewhere ?

Reply:
The wrapper is not copied from somewhere. just abstracted out redundant code from `commitAssemblerTransactions`